### ISSUE TITLE
Lost all items in dank

### ISSUE THREAD

**Issue by Momant24** [Iskallia/Vault-public#1055]
**Describe the bug**

**To Reproduce**
1. Upgrade dank
2.
3.
4.

**Expected behavior**

**Screenshots or Crash Report**
If applicable, add screenshots or crash reports to help explain your problem.

**Info (please complete the following information):**
- World Type: [e.g. Single Player or Server]
- The_Vault Version [e.g. 1.0.1]

**Additional context**

**Comment by fyrye** [comment]
How did you upgrade the dank, did you use the `dankstorage:dank_2` recipe in the crafting table or `dankstorage:1_to_2` in the dock?
If the first, that's why the `1_to_2` recipe exists, to ensure the original dank (and it's items) are not consumed.

**Comment by Momant24** [comment]
dank_2 recipe in the crafting table

Sendt fra Martin sin IPhone 13 mini

> 4. sep. 2022 kl. 16:24 skrev fyrye ***@***.***>:
> 
> dank_2 recipe in the crafting table

**Comment by fyrye** [comment]
Yea, this is not a bug related to `The_Vault`, more so a function of the Dank Storage mod. 

As of Dank Storage 1.12 they added the upgrade feature to not break the original dank, which is included in the VH modpack.
See below:
![image](https://user-images.githubusercontent.com/302306/188318986-23d0dd0a-be6a-4b4c-97c4-4c588470fde2.png)

**Comment by SpokenWig620933** [comment]
Actually due to the Dank_2 Recipe being custom with crafttweaker, this is the reason that the nbt data does not transfer with the upgrade, as such losing the items.

**Comment by fyrye** [comment]
Yes, I posted to provide an alternative where the end-result did not cause loss of items. You should link your fix @RedstoneGamer123 as an option for a alt as well.

Looking back I misworded my original posts, as it is a known issue with CraftTweaker and not related specifically to `The_Vault` mod, but more so of the VH modpack using CraftTweaker recipes that need additional changes to the scripts for many of the changed portable storage item recipes, like personal chests from mekanism (unusable in the Vault), Danks, and Shulker boxes (including dying them different colors).

**Comment by SpokenWig620933** [comment]
Here's a fix for the recipes causing the lost items, until the dev team adds it https://github.com/Iskallia/Vault-public-S1/issues/1069#issue-1365066270

**Comment by fyrye** [comment]
Yep, perfect! Except add context, "Here's a fix for the recipes causing the lost items, until the dev team adds it #1069".

**Comment by SpokenWig620933** [comment]
There :p
